features/features.go

```diff
@@ -124,6 +126,8 @@ var featureNames = map[SnapdFeature]string{
 	ConfdbControl: "confdb-control",
 
 	AppArmorPrompting: "apparmor-prompting",
+
+	GPIOChardevSupport: "gpio-chardev-support",
 }
 
 // featuresEnabledWhenUnset contains a set of features that are enabled when not explicitly configured.
```

Comment:
there's a `featuresExported` which I can't comment on as it's too far from the changed chunks of code, but unless flags cleanup is a problem, I would try to make this future proof and export a flag too, just in case we need it

Reply:
good point, updated